proto/lib/python/range.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package syft.lib.python;
+import "proto/core/common/common_object.proto";
+
+message Range {
+  int64 start = 1;
+  int64 stop = 2;
+  int64 step = 3;
+  bool has_start = 4;
```

Comment:
I think this has been introduced in Slices, but it's not necessary at all. You can validate if a field is Set with the `HasField` method. Could you address this and make the change in slices as well?

Reply:
Sure, will do.